components/MailchimpForm.tsx

```diff
@@ -57,14 +57,27 @@ function CustomForm(props: Props) {
     )
   }
 
+  const checkSubscribed = new RegExp('already subscribed', 'g')
+  const returnMessage = String(props.message)
```

Comment:
`props.message` can be either `undefine` or `Error`
When we wrap with `String(...)` it's will produce unexpected results.
 - `String(undefined)` will yield `"undefined"` => `"undefined".test(returnMessage)`
- `String(new Error(''))` will yield `"Error"` => `"Error".test(returnMessage)`

```suggestion
  const returnMessage =  typeof props.message === 'string' ?  String(props.message) : ''
```